src/vue-components/components/editElementHeader.vue

```diff
@@ -22,22 +22,28 @@
 </template>
 
 <script>
-
-    import {GridElement} from "../../js/model/GridElement.js";
+    import { GridElement } from "../../js/model/GridElement.js";
 
     export default {
         props: ["header", "gridElement", "openHelpFn", "closeFn"],
         data() {
             return {
-                GridElement: GridElement
+                GridElement: GridElement,
+                isEditing: false // Assuming there's a way to determine the editing mode
             }
         },
-        methods: {
-        },
-        mounted() {
-        },
+        computed: {
+            editingClass() {
+                return this.isEditing ? 'editing-on editing-on-cursor' : 'editing-off';
+            },
+            helpTooltip() {
+                return this.isEditing ? 'Click to get help in editing mode' : 'Click to get help in view mode';
+            },
+            closeTooltip() {
+                return this.isEditing ? 'Click to exit editing mode' : 'Click to close';
+            }
+        }
     }
 </script>
 
-<style scoped>
```

Comment:
CSS only for this compontent should be directly added here, not somewhere in an external .css file